index.html

```diff
@@ -22,6 +22,16 @@
                 <i class="fa fa-bars"></i>
             </a>
         </nav>
+        <nav id="mobile-nav-menu" class="mobile-menu">
+            <a class="close-button">
+                <i class="fa fa-close"></i>
+            </a>
+            <ul class="mobile-menu-items"> 
+                <li><a class="pop-nav" href="#portfolio">Portfolio</a></li>
+                <li><a class="pop-nav" href="#about">About</a></li>
+                <li><a class="pop-nav" href="#contact">Contact</a></li>
+             </ul>
+        </nav>
         <nav class="desktop-nav">
             <a class="logo-desktop">
                 My Logo
```

Comment:
- [ ] [OPTIONAL] Let's make sure that the logo and the h1 heading are on the same line. I tried to achieve this by decreasing the padding-left of the logo to 13px.
- [ ]  [OPTIONAL] I am also suggesting increasing the space between the navigation and the h1 heading. You can achieve this by adding a margin bottom of 50px or greater to the header like - ` header { margin-bottom: 50px; }`

![image](https://user-images.githubusercontent.com/4247740/161579499-94264f7e-11c3-4133-a3df-b3c551814fe4.png)

Reply:
@Cathella Thank you for the optional changes you suggested. Really appreciate that, it helped me make my portfolio website look better.